client/me/help/help-contact-closed/index.jsx

```diff
@@ -0,0 +1,79 @@
+
+export default localize( ( props ) => {
+	const {	translate } = props;
+
+	return (
+		<div className="help-contact-closed">
+			<FormSectionHeading>{ translate( 'Limited Support This Week' ) }</FormSectionHeading>
+			<p>
+				{ translate(
+					'Private email and live chat support will be closed from %(closed_start_date)s through %(closed_end_date)s, ' +
+					'included. We will reopen private email support on %(email_open_date)s, and live chat support will be back as ' +
+					'usual on %(livechat_open_date)s.', {
+						args: {
+							closed_start_date: 'Wednesday, September 14th, 2016',
+							closed_end_date: 'Wednesdaty September 23rd, 2016',
+							email_open_date: 'Thursday, September 22nd',
+							livechat_open_date: 'Monday, September 26th',
+						}
+					}
+				) }
+			</p>
+			<p>
+				{ translate(
+					'Why? Once a year, the WordPress.com Happiness Engineers and the rest of the WordPress.com family get together ' +
+					'to work on improving our services, building new features, and learning how to better serve you, our users. ' +
+					'But never fear! if you need help in the meantime:'
+				) }
+			</p>
+			<HelpContactClosedDetail icon="help">
+				{ translate(
+					'The {{link}}forums{{/link}} remain open and staffed during this time, ' +
+					'and you can post a support request using the form below.', {
+						components: {
+							link: <a href="https://en.forums.wordpress.com/" target="_blank" rel="noopener noreferrer" />
+						}
+					}
+				) }
+			</HelpContactClosedDetail>
+			<HelpContactClosedDetail icon="credit-card">
+				{ translate(
+					'If you require a refund, you can request one directly from your {{link}}Manage Purchases{{/link}} ' +
+					'screen.', {
+						components: {
+							link: <a href="/purchases" />
+						}
+					}
+				) }
+			</HelpContactClosedDetail>
+			<HelpContactClosedDetail icon="book">
+				{ translate(
+					'If you are new to WordPress.com we have a step-by-step {{guide_link}}guide{{/guide_link}} to all things WordPress. ' +
+					'You can find more details in our {{support_doc_link}}support documentation{{/support_doc_link}}. There we have ' +
+					'guides on {{get_started_link}}getting started{{/get_started_link}}, {{first_post_link}}writing your first ' +
+					'post{{/first_post_link}}, and {{find_readers_link}}finding your readers{{/find_readers_link}}.', {
+						components: {
+							guide_link: <a href="https://learn.wordpress.com/" target="_blank" rel="noopener noreferrer" />,
+							support_doc_link: <a href="https://support.wordpress.com/" target="_blank" rel="noopener noreferrer" />,
+							get_started_link: <a href="https://support.wordpress.com/start" target="_blank" rel="noopener noreferrer" />,
+							first_post_link: <a href="https://support.wordpress.com/create" target="_blank" rel="noopener noreferrer" />,
+							find_readers_link: <a href="https://support.wordpress.com/connect" target="_blank" rel="noopener noreferrer" />
```

Comment:
We could use `lib/url/support.js` to avoid hardcoding URLs here. I know the root URL is in there at least, the others might need to be added.

```js
import SUPPORT_URL from 'lib/url/support';
// ...
support_doc_link: <a href={ SUPPORT_URL.SUPPORT_ROOT } target="_blank" rel="noopener noreferrer" />
```